package.json

```diff
@@ -17,6 +17,7 @@
     "prepublishOnly": "npm run build:lib"
   },
   "devDependencies": {
+    "@angular-devkit/core": "0.0.28",
```

Comment:
why is this needed?

Reply:
When i tried to start the app. I got errors that this particular package is needed. When I installed this package locally everything went smooth. I could able to run the app locally on my computer. 
![image](https://user-images.githubusercontent.com/1368407/35334997-7a6ee762-0114-11e8-9aff-adbee32b5afa.png)

Reply:
okay, thanks for explanation. merging...